www/doc/2.0/api-endpoints.html

```diff
@@ -295,5 +295,55 @@ <h2 id="cost">/api/cost</h2>
     The contents of the points do not matter as long as they are
     in the correct format as mentioned above.</p>
 
+    <!-- Translate Endpoint -->
+
+    <h2 id="translate">/api/translate</h2>
+
+    <details>
+    <summary>Example inputs & outputs</summary>
+
+    <p>Request:</p>
+
+  <pre>{
+    formula: &lt;FPCore expression&gt;,
+    lang: "language"
```

Comment:
This might be a little bit late, but you should call this `language`, not `lang`, if possible. Avoid short names in code unless you are writing something like a mathematical equation. We have had autocomplete in editors for a long time now, and now we have that on super steroids, so it's fine to use full names.

Reply:
(We should notice here that this is a very basic way that the medium of a plain text file for representing code and the available tools can allow and force tradeoffs between the interests of the developer and the reader or user. As we think about language models, one avenue with lots of low-hanging fruit involves targeting those tradeoffs and trying to get the best of both worlds instead.)

Reply:
changed to "language"!